docs/products/clickhouse/get-started.md

```diff
@@ -5,77 +5,452 @@ sidebar_label: Get started
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-Start using Aiven for ClickHouse® by creating a service, adding a database, and connecting using [Docker](https://www.docker.com/).
+Start using Aiven for ClickHouse® by creating and configuring a service, connecting to it, and loading sample data.
 
-## Create a ClickHouse service
+## Prerequisites
 
-1.  Log in to the [Aiven Console](https://console.aiven.io/).
+Depending on a dev tool to use for working with Aiven for ClickHouse:
 
-1.  [Create an Aiven for ClickHouse® service](/docs/platform/howto/create_new_service).
+- Access to the [Aiven Console](https://console.aiven.io)
+- [ClickHouse CLI client](https://clickhouse.com/docs/en/install)
+- [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
+- [Aiven Operator for Kubernetes®](https://aiven.github.io/aiven-operator/installation/helm.html)
 
-    Once the service is ready, its status changes to **Running**, which typically takes a
-    couple of minutes, depending on your selected cloud provider and region.
+## Create a service
 
-## Create a database
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+[Create an Aiven for ClickHouse® service](/docs/platform/howto/create_new_service) in the
+[Aiven Console](https://console.aiven.io).
+</TabItem>
+<TabItem value="2" label="Terraform">
 
-Create a database either in the [Aiven Console](https://console.aiven.io/) or
-programmatically using the [Aiven API](/docs/tools/api).
+1. [Create an authentication token](/docs/platform/howto/create_authentication_token).
 
-:::important
-You cannot create a database using ClickHouse's SQL.
-:::
+1. Create the `sample.tf` file for the `aiven` provider configuration and
+   the `aiven_clickhouse` resource.
+
+    ```hcl
+    variable "aiven_api_token" {
+      type = string
+    }
+
+    variable "aiven_project_name" {
+      type = string
+    }
+
+    terraform {
+      required_providers {
+        aiven = {
+          source  = "aiven/aiven"
+          version = ">=4.0.0, <5.0.0"
+        }
+      }
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_api_token
+    }
 
-<Tabs>
-<TabItem value="1" label="Aiven Console" groupId="group1" default>
+    data "aiven_project" "sample" {
+      project = var.aiven_project_name
+    }
 
-1.  When the service is running, select **Databases and tables** from
-    the sidebar of your service's page.
+    resource "aiven_clickhouse" "clickhouse" {
+      project                 = data.aiven_project.sample.project
+      cloud_name              = "google-europe-west1"
+      plan                    = "startup-16"
+      service_name            = "my-clickhouse"
+      maintenance_window_dow  = "friday"
+      maintenance_window_time = "23:00:00"
+    }
+    ```
+
+1. Create the `terraform.tfvars` file for assigning actual values to your previously
+   declared variables.
+
+   ```hcl
+   aiven_api_token    = "AIVEN_API_TOKEN"
```

Comment:
nit: We are moving to calling users' tokens "personal tokens" and the UI doesn't use the term "API token" so just calling this `aiven_token` would make it consistent as well.